pkgs/development/python-modules/dearpygui/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, stdenv
+, python
+, cmake
+, glfw3
+, libX11
+}:
+
+let
+  # DearPyGui is not synced with last stable Imgui's version.
+  imgui = fetchFromGitHub {
+    owner = "ocornut";
+    repo = "imgui";
+    rev = "1b435ae3e07ca813eb3ef40aaabe7053f5570fae";
+    sha256 = "sha256-4UqRKgwaERUz2GQdEZbGmWOj8tAaP+thSlaOBRt06Ho=";
+  };
+in
+buildPythonPackage rec {
+  pname = "DearPyGui";
+  version = "v1.1.3";
```

```suggestion
  version = "1.1.3";
```
The version should start with a digit here, no "v" prefix. See [manual#sec-package-naming](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).